src/admin.c

```diff
@@ -384,6 +384,11 @@ master_key(char* password, bool generate_pwd, int pwd_length, int32_t output_for
       warnx("Could not write to master key file <%s>", &buf[0]);
       goto error;
    }
+      #if defined(HAVE_OSX)
+      #define GET_ENV(name) getenv(name)
```

Comment:
Please run `uncrustify.sh` before committing.

Also, should we rename the macro to `PGAGROAL_GETENV`?